doc/rtd/topics/debugging.rst

```diff
@@ -163,3 +163,63 @@ commandline:
 
 Inspect cloud-init.log for output of what operations were performed as a
 result.
+
+.. _proposed_sru_testing:
+
+Manually verifying cloud-init Stable Release Updates (SRU)
+==============================================================
+In order to retain stability on released Ubuntu series, cloud-init performs
+the following SRU procedure for `CloudinitUpdates`_.
+
+To manually test cloud-init **-proposed** versions
+before they are publicly released to **-updates** pockets:
+
+1. Launch a VM on your favorite platform, providing the cloud-config
+user-data `srutest.yaml`
+
+.. code:: bash
+
+    cat > srutest.yaml <<EOF
+    ## template: jinja
+    #cloud-config
+    ssh_import_id: [$LAUNCHPAD_USER]
+    hostname: SRU-worked-{{v1.cloud_name}}
+    EOF
+
+2. Wait for current cloud-init to complete
+
+.. code:: bash
+
+    ssh ubuntu@<YOUR_VM_IP> -- cloud-init status --wait
```

Comment:
If we had a `CI_VM_IP=<YOUR_VM_IP>` instruction earlier in the doc, then we could change all the `<YOUR_VM_IP>`s to `$CI_VM_IP` (or whatever variable name you think makes sense) and make the rest of the instructions much easier to copy/paste.